android/java/org/chromium/chrome/browser/preferences/BravePreferenceKeys.java

```diff
@@ -25,4 +25,18 @@
             "org.chromium.chrome.browser.Brave_Tab_Groups_Enabled";
     public static final String BRAVE_DISABLE_SHARING_HUB =
             "org.chromium.chrome.browser.Brave_Disable_Sharing_Hub";
+    public static final String BRAVE_NEWS_CARDS_VISITED = "brave_news_cards_visited";
```

Comment:
I cannot find where these are used in the project:
`BRAVE_NEWS_CARDS_VIEWED`
`BRAVE_NEWS_PROMOTION_CARDS_VISITED`
`BRAVE_NEWS_PROMOTION_CARDS_VIEWED`
`BRAVE_NEWS_DISPLAYAD_CARDS_VISITED`
`BRAVE_NEWS_DISPLAYAD_CARDS_VIEWED`
`BRAVE_NEWS_PREF_TURN_ON_NEWS`

If this is by intent, they should be commented out with explanation.
If this is accidentally, they should be removed.

Reply:
removed

Reply:
thanks